ospd_openvas/daemon.py

```diff
@@ -1157,6 +1162,8 @@ def exec_scan(self, scan_id: str):
 
                 # clean main_db, but wait for scanner to finish.
                 while not kbdb.target_is_finished(scan_id):
+                    if not self.is_openvas_process_alive(openvas_process):
+                        break
```

Comment:
```suggestion
                while not kbdb.target_is_finished(scan_id) and self.is_openvas_process_alive(openvas_process):
```

Reply:
I want to break the while only if the new if statement is not true.